### ISSUE TITLE
Unable to see Nickname

### ISSUE THREAD

**Issue by Skitcha** [Majekdor/HexNicks#22]
Hi there! i run a server with a handful of plugins, such as

- Vault
- Luckperms
- Brewery
- Worldedit
and then Hexnicks. But my issue is, the nicknames aren't showing up, i dont know if Im missing something in the config, where i can disable maybe a prefix or suffix, or if something is preventing me from showing the nick names. It shows that my nickname changed, but it only shows my username in the chat.

**Comment by Majekdor** [comment]
Are you using HexNicks to format chat? If so make sure chat-format is enabled in config and send me what your chat format is set to.

**Comment by Malvrius** [comment]
I have the same issue, I'm using Essentials Chat, And I even disabled the /nick command from essentials in their config and it didn't fix anything, I also disabled chat-format in the HexNicks config. I am using 1.17.1 paper.

Plugins (incase you need it):
Autorank, CustomCommands, EnhancedRegions, Essentials, EssentialsChat, EssentialsDiscord, EssentialsProtect, Factions, FakePrefix, FriendsSpigot, GCore, GoldenCrates, HexNicks, HolographicDisplays, HubThat, Infiniteannouncements, LPC, LuckPerms, Multiverse-Core, Multiverse-Inventories, Multiverse-NetherPortals, Multiverse-Portals, NexEngine, PlotSquared, ProBedwars*, ProtocolLib, ServerlistMOTD*, SetTheSpawn, SilkSpawners, TabList, Vault, VIPJoinMessage*, Votifier, VotingPlugin, WorldEdit, WorldGuard, WorldGuardDisableCollisions, WorldGuardEvents

**Comment by Majekdor** [comment]
Based on your plugin list, EssentialsChat is probably formatting your chat. I think you need to set change-displayname to false in the essentials config. If that doesn't work join my [discord](https://discord.makes.dev) because there are a lot of people who have had issues with essentials who may be able to help you.

**Comment by Malvrius** [comment]
That link sends me to "https://discord.makes.dev/", can you send me a new one?

**Comment by Malvrius** [comment]
That link sends me to "https://discord.makes.dev/", can you send me a new one?

**Comment by Majekdor** [comment]
Damn autocorrect on my phone. https://discord.majek.dev

**Comment by Skitcha** [comment]
> Are you using HexNicks to format chat? If so make sure chat-format is enabled in config and send me what your chat format is set to.

This is what my Config looks like

![image](https://user-images.githubusercontent.com/91161756/135535768-a50325f4-1c49-43c5-bccb-c1a9061bab7c.png)

**Comment by Malvrius** [comment]
I fixed it, I just uninstalled essentials chat and installed vaultchatformatter and set it up instead.

**Comment by Majekdor** [comment]
Yeah you need to set chat-format to true in the config if those 5 plugins are really all you have @Skitcha. None of the rest of those will format chat.
